salesagility/SuiteCRM
# Fix #8309 - Don't add delete action button without checking ACL Access

Delete action button is properly added with ACL Access checking in line 354 of the same file, then this second snippet that adds the same button with similar logic but without checking ACL Access should be removed.

## Motivation and Context
Fixes #8309 

## How To Test This
1. With a user without permissions for delete action, open a ListView.
2. Click search button.
3. When data appears, select any record using the checkboxes at the left side of the first column.
4. Delete action button is not shown.

## Types of changes


### Final checklist
